api/src/main/java/jakarta/data/repository/CrudRepository.java

```diff
@@ -77,18 +95,19 @@ public interface CrudRepository<T, K> extends BasicRepository<T, K> {
      *
      * <p>Non-matching entities are ignored and do not cause an error to be raised.</p>
      *
-     * @param entity the entity to update.
      * @return true if a matching entity was found in the database to update, otherwise false.
+@return true if a matching entity was found in the database to update, otherwise false.
```

Comment:
I'm sorry, I had an error in my prior suggestion, which accidentally caused the `@return` to be duplicated. This will fix it:
```suggestion
     * @param entity the entity to update. Must not be {@code null}.
     * @return true if a matching entity was found in the database to update, otherwise false.
```